### PR TITLE
Debug helpers

### PR DESCRIPTION
**Pull request by alexwdaniel**

### CODE REVIEW

File: test/wallaby/dsl_test.exs [hunk 1]
```diff
@@ -166,4 +166,24 @@ defmodule Wallaby.DSLTest do
...
   end
+
+  test "taking a screenshot", %{session: session, server: server} do
+    path =
+      session
+      |> visit(server.base_url)
+      |> take_screenshot
+
+    assert File.exists? path
+    File.rm_rf! path
```
Comment (alexwdaniel) [inline test/wallaby/dsl_test.exs]: It'd be nice to cleanup the /screenshots directory as well but I wasn't sure how to get the app root

Comment (alexwdaniel) [inline test/wallaby/dsl_test.exs]: Oh it should just be File.cwd

File: lib/wallaby/dsl/helpers.ex [hunk 2]
```diff
@@ -0,0 +1,43 @@
+defmodule Wallaby.DSL.Helpers do
+  alias Wallaby.Session
+
+  def take_screenshot(session) do
+    image_data =
+      Session.request(:get, "#{session.base_url}session/#{session.id}/screenshot")
```
Comment (keathley) [inline lib/wallaby/dsl/helpers.ex]: It doesn't need to happen in this PR but we should really abstract this away somehow.

**Comment by keathley** [comment]
This looks good to me overall. Should the screenshots directory be configurable in some way? Or does that probably not matter?

**Comment by alexwdaniel** [comment]
It's configurable in hound. It might be nice, but i don't really care where they go as long as they're not in the project root. I think it's ok for now.

**Comment by keathley** [comment]
I'm fine with it as is.

**Pull request merged**